tests/parser/integration/ParseKEYWORD.cpp

```diff
@@ -697,8 +697,8 @@ BOOST_AUTO_TEST_CASE( PORO_PERMX ) {
     BOOST_CHECK_EQUAL( 0.251224369 , poro[1]);
     BOOST_CHECK_EQUAL( 0.155628711 , poro[439]);
 
-    const std::vector<double>& permx = kw2.getSIDoubleData();
-    const std::vector<double>& permxRAW = kw2.getRawDoubleData();
+    auto permx = kw2.getSIDoubleData();
```

Comment:
Here you switch from using a reference to taking copy. It should still be ok using references as long as you do not mutate the underlying vector by calling the other get variant. Do you think that is something we need to guard against? If we need to take more copies than before that could reduce the benefit of this change.

Reply:
> Do you think that is something we need to guard against?

No I don't think this; the new possibility of getting this wrong is if you hold to a reference, and then a subsequent get changes the underlying data. That is of course a bit ugly - in particular when it comes about due to `get()` calls. However this is how C++ references work - for good and for bad; the main reason I am not worried is that the `Deck` should really not be used outside opm-common. This has inspired me revitalize the effort of getting reducing the deck usage.